frontend/storybook/src/components/entry/home/sections/aboutus-paragraph.ts

```diff
@@ -1,95 +1,83 @@
-import {argsToAttrs, deleteNone} from "@utils/attributes";
-import "@elements/entry/home/TOSORT/icon-wparagraph";
-import "@elements/entry/home/sections/aboutus-section";
+ import "@elements/entry/home/sections/aboutus-section";
```

Comment:
story name should match element name if that's the purpose. if "aboutus-paragraph" is something different than "aboutus-section" then I suppose it can be left as-is